ui/css/cloudstack3.css

```diff
@@ -12372,6 +12372,14 @@ div.ui-dialog div.autoscaler div.field-group div.form-container form div.form-it
   background-position: -35px -707px;
 }
 
+.shareTemplate .icon {
+  background-position: -165px -122px;
+}
+
+.shareTemplate .icon {
```

Comment:
@anurag should this be` .shareTemplate:hover .icon {`?

Reply:
Done.